optout/src/main/java/gov/cms/ab2d/optout/OptOutProcessorImpl.java

```diff
@@ -27,16 +27,23 @@
     @Override
     @Transactional
     public void process() {
-        log.info("import opt-out data - start ...");
 
-        try (var inputStreamReader = s3Gateway.getOptOutFile();
-             var bufferedReader = new BufferedReader(inputStreamReader)
-        ) {
+        final List<String> filenames = s3Gateway.listOptOutFiles();
+        filenames.stream().forEach(fileName -> fetchOptOutFile(fileName));
+
+    }
+
+    private void fetchOptOutFile(final String filename) {
+        try (var inputStreamReader = s3Gateway.getOptOutFile(filename);
+             var bufferedReader = new BufferedReader(inputStreamReader)) {
+
+            log.info("importing [{}]", filename);
+
             importOptOutRecords(bufferedReader);
 
-            log.info("import opt-out data - DONE.");
+            log.info("[{}] - import completed successfully", filename);
         } catch (IOException e) {
-            log.info("import opt-out data - FAILED.");
+            log.error("[{}] - import FAILED", filename);
             throw new UncheckedIOException(e);
```

Comment:
Now that we have multiple files, a failure of a single file should not cancel the entire import process: other files should still be attempted to process. I guess we can just log the stack trace and remove the `throw`

Reply:
done